### DISCUSSION TITLE
After Osu!Lazer is complete, what will the previous Osu!Lazer rankings and replays be like?

### DISCUSSION THREAD

**Discussion by huyenden** [ppy/osu#24225] [Q&A] [answered]
I have a series of questions regarding this issue:
- Will achievements from A to Sliver SS in Osu!Lazer be recorded in player profiles after Osu!Lazer and Score v2 are completed?
- Can replays being stored in memory be uploaded to Bancho if there is a leaderboard in the future? Because currently Lazer does not record replays.
- If uploading replays is allowed, is there a way to prevent replay editing (cheat) after uploading?
- With the ability to adjust the speed of Double Time and Nightcore from x1.5 to x2.0. Does Ppy intend to encourage players to rank at the x1.6 x1.7 x1.8 x1.9 x2.0 milestones? Questioning this because I think that in the future, high ranked players after reaching S HD DT will not feel secure when they feel they have to return to that HD DT map if someone reaches HD DT x1.6. Up to 26k ranked maps. Just a question about player psychology.
My recommendation is not to increase the DT/NC score factor beyond x1.5. Because that will respect the efforts of players 16 years ago who played in the Stable version.
- Will fun mods have their own difficulty coefficient in the future? Or will be reminded not to rate, as some mods like Magnetised, Wind down and Synesthesia make beatmaps a lot easier to go, especially when some mods amplify the difficulty.

**Comment by peppy** [comment, marked as answer]
We have mentioned many times that leaderboards will be reset once everything is in a good state. You can read more about the future by following community meetings or watching the changelog videos.
